### DISCUSSION TITLE
How can the latest changes be tried during the development?

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#7555] [Q&A]
Hi,

After making some changes to pixijs, how can I try these changes and see the result of how pixi renders after these changes?
`npm start` starts a server. But I am not able to figure out the url for pixi (dev) ..

Thanks

**Comment by ivanpopelyshev** [comment]
`https://pixijs.download/dev/pixi.js` is our testing url.

**Reply by a-c-sreedhar-reddy** [reply]
Hi @ivanpopelyshev 
I have some local changes and wanted to test them.

**Reply by ivanpopelyshev** [reply]
`npm run build`,  take contents of `bundles/pixi.js/dist` or `lib` , im not sure if `npm link` works there automatically, you can copy files. I personally use only umd build when developing, i dont need all that npm stuff

**Reply by ivanpopelyshev** [reply]
you can also clone pixi examples, and put `pixi.js` in `dist` folder of it - that way yo'll be able to use local version of pixi in examples.
